### ISSUE TITLE
Can't get pt_BR working

### ISSUE THREAD

**Issue by anselmobd** [heroku/heroku-buildpack-locale#13]
I try in many ways. None work.
I need pt_BR to use in Python/Django.
Locally I do:
```
locale.setlocale(locale.LC_ALL, 'pt_BR.utf8')
print('{:n}'.format(1230))

```
and get

> 1.230

with the correct thousand separator (".").

In Heroku:
I create a ".locales" file, put 

> pt_BR.utf8

inside it, and run
```
heroku buildpacks:add https://github.com/heroku/heroku-buildpack-locale
```
It don't work.

Than I did try numerous ideas:

- I came to heroku.com and put that buildpack above heroku/python
- I change the content of ".locales" to: pt_BR, pt_BR.UTF-8, pt_PT...
- I change the source code: "setlocale" to the same locale code and to others in many combinations

Never works.

Also never show what I expected when running the command:
```
heroku run "locale -a"
```
Please, give me some light.
Thanks in advance.

**Comment by andrecamara** [comment]
did you find a solution to this issue? same situation here

**Comment by tuanardouin** [comment]
Same issue. Opened a ticket with them, will update with the response.

**Comment by sandrofolk** [comment]
I have the same problem!

**Comment by tuanardouin** [comment]
I still can't change the locale but there are a couple of other techniques you may be able to try :

- The .profile file

A .profile file in the root of the project will be run during startup. Adding `export LANG=${LANG:-fr_FR}` may get that var set.

- The Procfile

A Procfile in the root of the project defines the processes to be run. Adding the export before the command may be a workaround? `web: LANG=fr_FR command-to-run`

**Comment by sandrofolk** [comment]
> I still can't change the locale but there are a couple of other techniques you may be able to try :
> 
> * The .profile file
> 
> A .profile file in the root of the project will be run during startup. Adding `export LANG=${LANG:-fr_FR}` may get that var set.
> 
> * The Procfile
> 
> A Procfile in the root of the project defines the processes to be run. Adding the export before the command may be a workaround? `web: LANG=fr_FR command-to-run`

it didn't work for me

**Comment by hoffmann04** [comment]
I have the same issue! Everything works fine in my localhome project, with:

import locale
locale.setlocale(locale.LC_ALL, 'pt_BR')

But when I try to deploy to heroku, it always crash. And using the buildpack didn't work at all, it only show en languages when I run the locale -a command

**Comment by dmathieu** [comment]
Yes, `locale -a` will not work. You should set $LANG and type a localized command (such as `date`) to check it on the terminal.

This buildpack is using quite exotic system config variables to install new locales in user-land instead of `/usr`.
All it does is set those config vars and download the locale. If a specific locale doesn't work, it's probably either because it doesn't exist (though I believe we should error on build when that happens), or something is wrong with the way you're using it.

Feel free to investigate the buildpack's code if you want to better understand all this.
